src/extensions/replay/sessionrecording.ts

```diff
@@ -290,10 +294,6 @@ export class SessionRecording {
         return this.instance.config.session_recording.session_idle_threshold_ms || RECORDING_IDLE_THRESHOLD_MS
     }
 
-    private get rrwebRecord(): rrwebRecord | undefined {
```

Comment:
flyby change to make this a function instead of a class method
we can't minify class methods so every call to `rrwebRecord` uses the fullname

this PR was +85B and is now +15B

not much in the grand scheme of things but better than nothing